src/main/java/arona/AronaException.java

```diff
@@ -0,0 +1,7 @@
+package arona;
+
```

Comment:
Perhaps it would be good to describe the purpose of this class because it seemed a bit redundant at first glance.